lib/pbench/agent/results.py

```diff
@@ -62,18 +62,17 @@ def make_result_tb(self):
             )
             sys.exit(0)
 
+        mdlog_name = self.result_dir / "metadata.log"
+        mdlog = ConfigParser()
         try:
-            md_log = (self.result_dir / "metadata.log").resolve(strict=True)
+            with mdlog_name.open("r") as fp:
+                mdlog.read_file(fp)
         except FileNotFoundError:
             self.logger.debug(
                 "The {}/metadata.log file seems to be missing", pbench_run_name
             )
             sys.exit(0)
-
-        opts, _ = configtools.parse_args()
-        opts.filename = md_log
-        conf_md, _ = configtools.init(opts, None)
-        md_config = conf_md["pbench"]
```

Comment:
Well, OK; that's definitely squirrelly. Parse the standard Pbench command arguments, then punch the config object to point at the metadata.log instead of pbench-agent.cfg... clever, but smelly code.